src/xdp/xdp.vcxproj

```diff
@@ -137,7 +137,8 @@
       </AdditionalDependencies>
     </Link>
   </ItemDefinitionGroup>
-  <ItemDefinitionGroup Condition="'$(LatestWdkPlatformVersion)' &gt;= '10.0.26100.0'">
+  <ItemDefinitionGroup Condition="'$(LatestWdkPlatformVersion)' != '' And '$(LatestWdkPlatformVersion)' &gt;= '10.0.26100.0'">
+    <!-- a null check is needed to let nuget parse this before LatestWdkPlatformVersion is defined when packages are initially restored -->
```

Comment:
The OS repo has a slightly different check to determine whether a restore is happening (there's a property that gets set if and only if a restore is happening) but I think this will work fine.

Reply:
I'm happy to change it if there is a better solution that work on all systems - why reasoning is only that for a real build (when that group matter), `LatestWdkPlatformVersion` would have to be set, or there is no wdk, which would be a problem by itself.

Reply:
The check I was thinking of is `'$(RestoreSolutionDirectory)' != ''` implying a restore is in progress.